src/createPrevArgSaver.js

```diff
@@ -14,8 +14,14 @@
  *
  * @param arg
  */
+
 function createPrevArgSaver(arg) {
-  // write code here
+  let prevValue;
+  return (current) => {
```

Comment:
для одного аргумента скобки не нужны
```suggestion
  return current => {
```